src/Http/ServerRequest.php

```diff
@@ -1258,6 +1261,11 @@ public function getData(?string $name = null, $default = null)
      */
     public function input(?callable $callback = null, ...$args)
     {
+        deprecationWarning(
```

Comment:
I am having second thoughts about this explicit warning, it can be annoying to people. The `@deprecated` tag will already warn IDE users.

Reply:
This won't show up for more people unless they are calling it though.